cljfmt/src/cljfmt/main.clj

```diff
@@ -111,34 +113,40 @@
   ([paths]
    (check paths {}))
   ([paths options]
-   (let [counts (transduce
-                 (comp (mapcat (partial find-files options))
-                       (map (partial check-one options))
-                       (map (fn [status]
-                              (print-file-status options status)
-                              (:counts status))))
-                 (completing merge-counts)
-                 paths)]
+   (let [map* (if (:parallel? options) pmap map)
```

Comment:
Can you align the `let` terms? So:

```clojure
(let [map*   (if (:parallel? options) pmap map)
      counts (->> paths
```

Reply:
Aye, done. I'll try to keep that in mind.